common/run/v1alpha/run.proto

```diff
@@ -25,3 +28,26 @@ enum RunSource {
   // Run from API or SDK.
   RUN_SOURCE_API = 2;
 }
+
+// TimeRangeType defines the time range of a trend.
+enum TimeRangeType {
```

Comment:
Setting this feels a bit off to me. The backend should accept any value within a range, and handling enums should be a frontend task.

Reply:
➕ I strongly agree. We should let the client specify the aggregation window and the start / end values instead of hiding these query values under a time range.